model/resolver/LinkedDataResource.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+class LinkedDataResource implements RemoteResource
+{
+    private $uri;
+
+    public function __construct(string $uri) {
+        $this->uri = $uri;
+    }
+    
+    public function resolve(int $timeout) : ?EasyRdf\Resource {
+        // prevent parsing errors for sources which return invalid JSON (see #447)
+        // 1. Unregister the legacy RDF/JSON parser, we don't want to use it
+        EasyRdf\Format::unregister('json');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'EasyRdf\Format' in method 'resolve'.](https://app.codacy.com/manual/NatLibFi/Skosmos/pullRequest?prid=4951447)